client/my-sites/people/people-profile/index.jsx

```diff
@@ -275,12 +275,11 @@ class PeopleProfile extends Component {
 }
 
 export default connect( ( _state, { siteId, user } ) => {
-	const userId = user && user.ID;
-	const linkedUserId = user && user.linked_user_ID;
+	const hasWPCOMAccountLinked = false !== user?.linked_user_ID;
 	const externalContributors = ( siteId && requestExternalContributors( siteId ).data ) || [];
 	return {
 		isExternalContributor: externalContributors.includes(
-			undefined !== linkedUserId ? linkedUserId : userId
+			hasWPCOMAccountLinked ? user?.linked_user_ID || user?.ID : user?.ID
```

Comment:
Nitpick: Found a bit hard to understand this. I think the code below should work too and looks simpler?
```suggestion
				user?.linked_user_ID ?? user?.ID
```

Reply:
Good call, changed it 8927c1b